matchms/similarity/__init__.py

```diff
@@ -21,6 +21,7 @@
 from .IntersectMz import IntersectMz
 from .ModifiedCosine import ModifiedCosine
 from .ParentmassMatch import ParentmassMatch
+from .PrecursormzMatch import PrecursormzMatch
```

Comment:
The doc string above lists a couple of matcher classes. Maybe nice to add `PrecursormzMatch` next to `ParentmassMatch` class.

Reply:
Good point. Added it to the docstring.

Reply:
thanks